### DISCUSSION TITLE
Read contents of an linux image, created with dd

### DISCUSSION THREAD

**Discussion by xennialex** [adoconnection/SevenZipExtractor#73] [Q&A]
Hi!

I managed to open the first part of the BIN file like this:
`using ArchiveFile archiveFile = new ArchiveFile(stream, SevenZipFormat.Mbr);`

But now I can't find a solution to open the 0.img (that's the only file entry).
Is there a SevenZipFormat for an linux ext file system? 

7-zip itself can read the file just fine and is showing "Linux" as filesystem

**Comment by adoconnection** [comment]
Hi, I guess you should try make it in two steps, like tar.gz
https://github.com/adoconnection/SevenZipExtractor/wiki/Extract-tar.gz,-tag.xz

**Reply by xennialex** [reply]
I tried implementing it like this, but I couldn't find a SevenZipFormat that would work with an ext file system (and I tried ALL from that enum 😅).

I did a similar implementation with a MBR and FAT file system that works just fine, so my only problem is the format I should use for ext...
